platforms/php/versionsToBuild_11172020.txt

```diff
@@ -0,0 +1,15 @@
+# version, sha, gpg keys, dockerfile
```

Comment:
I did not get what is this file for vs. the earlier file

Reply:
yeah ... so this file I created so that I can build latest sdks of python and php to limit the scope (i.e. only what i need), earlier it was trying to build all the versions and some of them (read unsupported old version e.g. php 5, php 7, python 2 etc) was throwing error, so I got rid of them by creating a new version of this file. From now onward we will probably not be needing to build old sdks (which is already present in storage account) of stretch and only be adding future versions. So, we may not be breaking anything. But I still kept an copy  in case we need to refer old sdks etc.